composer.json

```diff
@@ -32,8 +32,8 @@
     "oat-sa/extension-tao-community": "10.2.0",
     "oat-sa/extension-tao-funcacl": "7.2.0",
     "oat-sa/extension-tao-dac-simple": "7.4.1",
-    "oat-sa/extension-tao-itemqti": "28.28.0",
-    "oat-sa/extension-tao-testqti": "42.0.0",
+    "oat-sa/extension-tao-itemqti": "dev-release-2022-01 as 28.28.1",
+    "oat-sa/extension-tao-testqti": "dev-release-2022-01 as 42.0.1",
```

Comment:
I know it won't really matter in the end, when installing the package, but could you please update the `.lock` file as well?
It's still useful to do so in order to make sure the compatible versions are set in `composer.json`.

Reply:
Sure, I will.
I'm waiting for the other PR to be merged, then I'll update it :)

Reply:
Companion PR merged, lock file updated ;)